indico/htdocs/sass/partials/_boxes.scss

```diff
@@ -407,10 +403,42 @@ $i-box-padding: 10px;
 
     > li {
         @include border-top();
-        display: block;
-        padding: 10px 10px;
+        padding: 10px;
         position: relative;
         overflow: hidden;
+
+        /* Sortable row with handle on the left */
```

Comment:
It's also possible to change the license header to use bang comments and will be ignored by the rule.
```
/*!
  * This is a good bang comment
**/
```
 I don't know if these bang comments have an specific use in SASS files though.